wallet-new/src/Cardano/Wallet/Kernel/BIP39.hs

```diff
@@ -0,0 +1,12 @@
+{-- | This module trivially re-exports everything from 'Pos.Util.Mnemonic',
+      in preparation for the final deprecation of 'wallet' code once the new
+      data layer will be delivered.
+--}
+
+module Cardano.Wallet.Kernel.BIP39 (
+    module LegacyCompat
+    ) where
+
+import           Pos.Util.Mnemonic as LegacyCompat
```

Comment:
@KtorZ A potential strategy we might adopt regarding our previous conversation on how to identify modules which needs migration.

Here I have "wrapped" the legacy stuff in a presentable module name which is re-export everything. If we follow the same pattern, it should be easy to search for "LegacyCompat" and gradually replace things up until we can decommission the `wallet` project 😀